src/app/shared/components/remove-account-success/remove-account-success.component.scss

```diff
@@ -0,0 +1,36 @@
+@import "src/assets/styles/vars";
+
+:host {
+  text-align: center;
+  font-family: var(--font-open-sans);
+}
+
+.icon {
+  color: var(--c-green);
+  font-size: 24px;
+
+  .material-icons-outlined {
+    font-size: 38px;
+  }
+}
+
+.title {
+  font-size: 24px;
+  font-weight: bold;
+  letter-spacing: -0.19px;
+  color: #052c65;
+  margin-bottom: 25px;
+}
+
+::ng-deep .dialog-remove-account-success .mat-dialog-container {
```

Comment:
ponoć ::ng-deep jest deprecated

Reply:
https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep